<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>10</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="PosCtrlTab">
     <attribute name="title">
      <string>PosCtrl</string>
     </attribute>
     <widget class="PosCtrl" name="PosCtrlWidget" native="true">
      <property name="geometry">
       <rect>
        <x>18</x>
        <y>5</y>
        <width>160</width>
        <height>160</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="RotCtrlTab">
     <attribute name="title">
      <string>RotCtrl</string>
     </attribute>
     <widget class="RotCtrl" name="RotCtrlWidget" native="true">
      <property name="geometry">
       <rect>
        <x>18</x>
        <y>5</y>
        <width>160</width>
        <height>160</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="MsgShow">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>215</y>
      <width>200</width>
      <height>141</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="ArmGL" name="RobotShow" native="true">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>370</y>
      <width>200</width>
      <height>190</height>
     </rect>
    </property>
   </widget>
   <widget class="JointPos" name="JointPosWidget" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>325</y>
      <width>525</width>
      <height>62</height>
     </rect>
    </property>
   </widget>
   <widget class="CmdSend" name="CmdSendWidget" native="true">
    <property name="geometry">
     <rect>
      <x>114</x>
      <y>400</y>
      <width>441</width>
      <height>156</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="RobotName">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>400</y>
      <width>50</width>
      <height>156</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/img/ExecutionName.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
   </widget>
   <widget class="QWidget" name="CameraFigure" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>525</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAnalyze">
    <property name="title">
     <string>Analyze</string>
    </property>
    <addaction name="actionJointPos_Feedback"/>
    <addaction name="actionGripper_Feedback"/>
    <addaction name="actionDeveloper_Mode"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_DRobot_Ctrl_Version"/>
    <addaction name="actionUpdate_Control_Firmware"/>
   </widget>
   <addaction name="menuAnalyze"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionJoint1">
   <property name="text">
    <string>Joint1</string>
   </property>
  </action>
  <action name="actionJoint2">
   <property name="text">
    <string>Joint2</string>
   </property>
  </action>
  <action name="actionJoint3">
   <property name="text">
    <string>Joint3</string>
   </property>
  </action>
  <action name="actionJoint4">
   <property name="text">
    <string>Joint4</string>
   </property>
  </action>
  <action name="actionJoint5">
   <property name="text">
    <string>Joint5</string>
   </property>
  </action>
  <action name="actionJoint6">
   <property name="text">
    <string>Joint6</string>
   </property>
  </action>
  <action name="actionGripperPosition">
   <property name="text">
    <string>Gripper Position</string>
   </property>
  </action>
  <action name="actionGripperVelocity">
   <property name="text">
    <string>GripperVelocity</string>
   </property>
  </action>
  <action name="actionGripperForce">
   <property name="text">
    <string>GripperForce</string>
   </property>
  </action>
  <action name="actionAbout_DRobot_Ctrl_Version">
   <property name="text">
    <string>About DRobot_Ctrl Version</string>
   </property>
  </action>
  <action name="actionJoint_Position_Feedback">
   <property name="text">
    <string>Joint Position Feedback</string>
   </property>
  </action>
  <action name="actionJoint_Velocity_Feedback">
   <property name="text">
    <string>Joint Velocity Feedback</string>
   </property>
  </action>
  <action name="actionJoint_Force_Feedback">
   <property name="text">
    <string>Joint Force Feedback</string>
   </property>
  </action>
  <action name="actionGripper_Feedback">
   <property name="text">
    <string>Gripper Feedback</string>
   </property>
  </action>
  <action name="actionDeveloper_Mode">
   <property name="text">
    <string>Developer Mode</string>
   </property>
  </action>
  <action name="actionUpdate_Motor_Firmware">
   <property name="text">
    <string>Update Motor Firmware</string>
   </property>
  </action>
  <action name="actionUpdate_Master_Control_Firmware">
   <property name="text">
    <string>Update Control Firmware</string>
   </property>
  </action>
  <action name="actionJointPos_Feedback">
   <property name="text">
    <string>Joint Position Feedback</string>
   </property>
  </action>
  <action name="actionUpdate_Control_Firmware">
   <property name="text">
    <string>Update Control Firmware</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PosCtrl</class>
   <extends>QWidget</extends>
   <header location="global">posctrl.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RotCtrl</class>
   <extends>QWidget</extends>
   <header location="global">rotctrl.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ArmGL</class>
   <extends>QWidget</extends>
   <header location="global">armgl.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>JointPos</class>
   <extends>QWidget</extends>
   <header location="global">jointpos.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CmdSend</class>
   <extends>QWidget</extends>
   <header location="global">cmdsend.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
